locales/en-US/editor.properties

```diff
@@ -116,6 +116,7 @@ CMD_FILE_NEW=New File
 CMD_FILE_NEW_FOLDER=New Folder
 CMD_FILE_RENAME=Rename
 CMD_FILE_DELETE=Delete
+CMD_FILE_DOWNLOAD=Download
```

Comment:
All you need to do is also add the same line to the `locales/en-US/editor.properties` file in the thimble repository and open up a PR for it. If you look closely, that file is identical to this one.